packages/documentation/pages/usage/components/field-inline-edit.vue

```diff
@@ -0,0 +1,194 @@
+<template>
+	<div>
+		<ComponentInfo v-bind="{ component }" />
+
+		<div class="element-example white">
+			<!-- <KtForm v-model="values"> -->
+			<KtFieldInlineEdit
+				v-bind="ktFieldInlineEditProps"
+				class="mb-16px"
+				:isEditing="isEditing"
+				:value="fieldValue"
+				@confirm="onConfirm"
+				@input="setValue"
+				@update:isEditing="($event) => (isEditing = $event)"
+			/>
+			<!-- formKey="fieldValue" -->
+			<!-- </KtForm> -->
+			<KtForm v-model="settings" size="small">
+				<div class="wrapper">
+					<div>
+						<h4>Settings</h4>
+						<KtFieldSingleSelect
+							formKey="validation"
+							isOptional
+							label="Validation State"
+							:options="validationOptions"
+						/>
+						<KtFieldSingleSelect
+							formKey="preventConfirmationOn"
+							isOptional
+							label="preventConfirmationOn"
+							:options="preventConfirmationOptions"
+						/>
+						<KtFieldSingleSelect
+							formKey="textStyle"
+							:isDisabled="settings.booleanFlags.isMultiline"
+							isOptional
+							label="Text Styles"
+							:options="textStyleOptions"
+						/>
+						<KtFieldToggleGroup
```

Comment:
Not sure what's up with this component, cannot interact with it.
Maybe using `KtFormControllerObject` to drill into `booleanFlags` will make it work?